redbot/core/core_commands.py

```diff
@@ -705,15 +705,14 @@ class Core(commands.Cog, CoreLogic):
         if ctx.invoked_subcommand is None:
             if ctx.guild:
                 guild = ctx.guild
-                admin_role = (
-                    guild.get_role(await ctx.bot.db.guild(ctx.guild).admin_role()) or "Not set"
-                )
-                mod_role = (
-                    guild.get_role(await ctx.bot.db.guild(ctx.guild).mod_role()) or "Not set"
-                )
+                admin_role_ids = await ctx.bot.db.guild(ctx.guild).admin_role()
+                admin_roles = [r for r in guild.roles if r.id in admin_role_ids]
+                admin_roles_str = humanize_list(admin_roles) if admin_roles else "Not Set."
+                mod_roles = [r for r in guild.roles if r.id in admin_role_ids]
+                mod_roles_str = humanize_list(mod_roles) if mod_roles else "Not Set."
                 prefixes = await ctx.bot.db.guild(ctx.guild).prefix()
-                guild_settings = _("Admin role: {admin}\nMod role: {mod}\n").format(
-                    admin=admin_role, mod=mod_role
+                guild_settings = _("Admin roles: {admin}\nMod roles: {mod}\n").format(
+                    admin=admin_roles_str, mod=mod_roles_str
```

Comment:
This section looks fine at a glance, but it warrants adding pagination to the send below to prevent something really stupid with large number of roles.